proposals/NightlyBuilds.md

```diff
@@ -0,0 +1,49 @@
+# Cordova nightly builds proposal

```

Comment:
I've started maintaining a TOC here: https://github.com/cordova/cordova-discuss#readme. It will be great to add this there as an "In Progress" proposal.